inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convert.cpp

```diff
@@ -1,47 +0,0 @@
-// Copyright (C) 2018-2021 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include <vector>
-
-#include "single_layer_tests/convert.hpp"
```

Comment:
What about "single_layer_tests/convert.hpp" ?

Reply:
Deleted thank you for noticing this.